react-debugging.html

```diff
@@ -65,11 +65,12 @@
         // No need to change anything ABOVE this line ^
 
         return (
-          <h1>Draw a Card</h1>
-          <button onClick={}>
+          <>
+        <h1>Draw a Card</h1>
+        <button onClick={drawCard}>
             Click to Draw a Card
           </button>
-          <button onclick={shuffle}>
+          <button onClick={hand}>
             Return Cards and Shuffle Deck
```

Comment:
So close! You just need to change the 'c' in onclick, but the  shuffle method is correct.